backend/data/processors/conditions_preprocessing.py

```diff
@@ -269,6 +310,24 @@ def convert_including(processed):
     return re.sub("including", "&&", processed)
 
 
+def convert_manual_programs_and_specialisations(processed):
+    """
+    Deals with the following cases:
+    - Enrolment in a x program
```

Comment:
Maybe using fstring syntax would make it a little clearer at first glance, e.g. Enrolment in a {processed} program. Up to you

Reply:
Yeah it's absolutely awful, but I'm not sure what you mean by making it an fstring since both strings are fstrings already?

Reply:
actually nvm